package.json

```diff
@@ -19,6 +19,7 @@
   "devDependencies": {
     "jest": "23.4.2",
     "jest-buble": "2.0.0",
+    "prop-types": "15.6.2",
```

Comment:
Do you need `prop-types` in the root package.json? Isn't it sufficient to have it in each package?

Reply:
For tests 👍